packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -337,43 +354,34 @@ $Calendar-dayPicker-margin: 10px;
   }
 }
 
-
- .pickerOnlyMonthPickerVisible {
-
-  .yearComponents,
-  .decadeComponents {
-    top: 9px;
-  }
- }
-
 // 460px and up.
 //
 // On screens that can fit it, we show the month picker next to the day picker if we have it enabled.
 @media (min-width: 460px) {
 
-    $Calendar-day: 30px;
+  $Calendar-day: 28px;
+  $Calendar-month: 40px;
 
-    // Update the spacing and text for the day and month pickers
+  // Update the spacing and text for the day and month pickers
 
-    .header {
-      height: $Calendar-day;
-      line-height: $Calendar-day;
-    }
+  .dayPicker,.monthPicker {
+    min-height: ( $Calendar-day * 8 ) - $Calendar-dayPicker-margin;
```

Comment:
Why do we need this math in the styles? More efficient to not have these computations